android/src/main/kotlin/com/lahaus/iterable_flutter/IterableFlutterPlugin.kt

```diff
@@ -107,11 +107,10 @@ class IterableFlutterPlugin : FlutterPlugin, MethodCallHandler {
         notifyPushNotificationOpened()
         false
       }
-
+    
     if (activeLogDebug) {
-      configBuilder.setLogLevel(Log.DEBUG)
+      configBuilder.setLogLevel(Log.VERBOSE)
```

Comment:
This is probably not necessary to include in the PR and probably makes sense to change locally when you need to check logs for something.